clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -2339,4 +2340,35 @@ describe("Consent overlay", () => {
         .should("have.length", 3);
     });
   });
+
+  describe("fides overrides", () => {
```

Comment:
```suggestion
  describe("fides experience translation overrides", () => {
```

nitpicky 😄

Reply:
hmm so I want to keep these as generic overrides since here I'm specifically testing `fides_primary_color`. In the `privacy-center/cypress/e2e/consent-i18n.cy.ts` is where I test the experience translation overrides through localization.